client/reader/discover/discover-stream.js

```diff
@@ -91,6 +93,18 @@ const DiscoverStream = ( props ) => {
 		selectedStreamName: selectedTab,
 	};
 
+	if ( selectedTab === 'add-new' ) {
+		return (
+			<ReaderMain className={ clsx( 'following main', props.className ) }>
```

Comment:
These classes doesn't seem to affect any styling so we can remove them.

```suggestion
			<ReaderMain className={ props.className }>
```

Reply:
There are some tab hover and [max-width styling](https://github.com/Automattic/wp-calypso/blob/trunk/client/reader/stream/style.scss#L8) based on these classes, so we can't remove them wholesale without some refactoring.

`<ReaderMain className={ clsx( 'following main', props.className ) }>` is basically reproducing what the Stream component renders [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/reader/discover/discover-stream.js#L95) for the rest of the tabs.